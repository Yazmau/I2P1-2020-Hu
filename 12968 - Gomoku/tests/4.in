30
...b......b....
..........b....
.......w.......
.w....b........
........wb..b.b
...............
.....bb........
...............
...............
...............
..............w
.....b.......w.
....w..........
....w..........
..w...ww.......
...............
........b......
...............
...........w...
b.........www..
..........wbw..
..........wbw..
...........b...
...........b...
...........b...
...........w..w
...............
...............
...............
...........bbbb
........b......
...............
..bbb..........
..bwwwwwb......
..bbb..........
......b........
...............
...............
...............
...............
...............
...............
.....w.........
.......w.......
............www
..b.....b......
.......b.......
...............
...w...........
.....b..b.w....
........w......
b..............
...w...........
...............
..............b
.....b.........
...b...........
........w......
...........bwww
w...........w.b
...............
............b..
...............
.............b.
...............
...............
...............
.........bbb...
..w......bwb...
.........bwb...
..........w....
..........w....
..........w....
.w........b....
............www
...............
...........b...
...............
...............
...............
...............
......bbb......
......bwwwwwb..
......bbb......
...............
..........w....
...............
............b..
..w............
............www
.....b.........
..b............
...............
.....b.........
...bw..bb..w...
w..............
.............b.
.....b..w......
...........w...
...w...........
...............
............w..
....b.......b..
.w.............
.w......w......
...............
b..............
.b.............
...............
...............
.......bbb.....
.......bwb.w...
.......bwb.....
........w......
........w......
........w......
........b....w.
...............
...............
............www
.b.............
...............
...............
b..............
...............
...............
...............
...............
..w............
.............w.
...............
..www..........
..wbbbbbw......
..www..........
...........bbbb
.............bb
...............
.............b.
........w...b..
....b..........
......w.......w
.....b.........
..........b....
........w......
...........b...
....b...w....b.
........w......
......w........
.w..........w..
...w...........
........b......
...............
............b..
...............
...bbb.........
...bwb.........
...bwb.........
....w.......w..
....w..........
....w..........
..w.b..........
...............
...............
...............
............www
...............
..b............
...............
...............
.www........b..
.wbbbbbw.......
wwww...........
...............
...............
...............
...............
.....w.........
...............
...............
...........bbbb
..b............
.....b.........
.b.............
.............b.
.w....bb...w...
.b............b
.............w.
..b............
...............
...b...........
...............
w..............
....w....w....w
..............w
.........ww...b
...............
...............
...b...........
...............
....www........
....wbw........
....wbw........
.....b.........
.....b.........
.....b.b.......
....ww.........
...............
...............
...............
......w....bbbb
...............
...............
........b......
...............
..w............
...............
.....www.......
.....wbbbbbw...
.....www.......
...............
.........b.....
...............
.w.............
...............
...........bbbb
...........b..b
......b........
.b.............
...............
..........w....
.b....w........
..............b
........b..w...
.b........w....
..b............
.....w.........
.w.....b.......
..............w
.w.........w...
.............w.
...............
...............
...............
...............
bbb............
bwb...b........
bwb............
.w.............
.w..b..w.......
.w............w
.b.............
...............
...............
...............
............www
...............
...............
..b.....bbb....
...w....bwwwwwb
........bbb....
...............
...............
...............
...............
...............
...............
...............
.........b.....
............w..
............www
...bb....b..b..
.........b.....
...............
.....b........b
...............
.....w......w..
...............
.........bw....
..b............
..w..ww.......b
.w....w........
.......w.......
...............
...............
.......w.......
...............
...............
...............
...............
.www...........
bwbw...........
.wbw...........
..b....b.......
..b............
..b............
..w........w...
...............
........w......
...............
...........bbbb
...............
...............
........b......
...............
.......b.......
........w......
...............
...............
...............
........www....
........wbbbbbw
........www....
...............
.........w.....
...........bbbb
.............b.
....b.b.....w..
...............
.......b.......
...............
b....w....w....
....w..........
.b.............
.b......b...w..
..............w
.wb............
......w........
..............b
...............
.....w.w......b
...............
www............
wbw............
wbw.......b....
.b...........b.
.b.............
.b.............
.w.............
...............
...............
...............
w..............
...............
...............
.....w.....bbbb
...............
...............
...............
...............
...............
..bbb.......b..
..bwwwwwb......
..bbb..........
...............
............b..
..........w....
.......w.......
...............
...............
............www
...............
...........bb..
.....b.........
.....b.........
.......b.....b.
...............
.....b..b......
.w..........w..
...............
.w.......b....b
..........w.w.w
w.w.......w....
...........w...
...............
..............b
...............
...............
...............
..........b....
...............
.www...........
.wbw...........
.wbw...........
..b.......b.w..
..b............
..b............
..w....w.......
...............
...............
...........bbbb
...............
...............
...............
...............
...............
...............
...............
.......bbb.....
......bbwwwwwb.
.......bbb.....
b..............
...............
...............
...w...........
..........w.www
......b........
...............
............b..
.b..........b..
.............b.
..b............
.......w.......
.....ww....w...
..........b....
.....w.........
..b.......wb...
...w....w...b..
..............w
...............
......w.......b
...........b...
...............
...............
...............
...............
.......www..w..
.......wbw.....
.......wbw.....
.......bb......
........b......
........b......
........w......
..w............
...............
...........bbbb
....b..........
...............
...............
...............
...............
...............
...............
.....w.........
.......bbb.....
.......bwwwwwbb
.......bbb.....
...............
...............
...............
w...........www
