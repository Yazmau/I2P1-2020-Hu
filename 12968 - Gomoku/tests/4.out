none
black
white
none
white
white
none
white
black
none
white
black
none
black
black
none
white
white
none
black
black
none
black
white
none
black
white
none
black
white
